Mathlib/RingTheory/Generators.lean

```diff
@@ -112,31 +112,52 @@ def ofSurjective {vars} (val : vars → S) (h : Function.Surjective (aeval (R :=
   σ' x := (h x).choose
   aeval_val_σ' x := (h x).choose_spec
 
+/-- Let lean see through the `vars` field to enable the `MvPolynomial` API. -/
+unif_hint ofSurjective_vars_eq {vars} (val : vars → S)
+    (h : Function.Surjective (aeval (R := R) val)) where
+  ⊢ (ofSurjective val h).vars ≟ vars
```

Comment:
Can you put the PR message (or something similar) in a library note and just link to that in these `unif_hints`?